### ISSUE TITLE
Tests.

### ISSUE THREAD

**Issue by zachriggle** [crackinglandia/pype32#6]
There are no tests.  I'd recommend looking into a corpus of PE files and using Travis CI, Shippable, or Circle Ci to ensure that all of the files parse correctly.  In particular, it's probably worth getting some really old (16-bit) and really new (Windows 10), and relatively uncommon (Windows on ARM) files to make sure that everything parses.

**Comment by poxyran** [comment]
Yes, my bad, no testcases were created. However, I did some manual tests using the PE corpus from corkami and fixed some bugs. Anyway, that not the right way to do it, I know.

I didn't know about Travis CI, I'll take a look. 

Btw, if you have binaries from Windows on ARM, please, send them to me.

**Comment by zachriggle** [comment]
For things you don't have any of, it may be easiest to just compile "hello world" with a recent Visual Studio Express (free as in beer).
